hal/armv7a/imx6ull/timer.c

```diff
@@ -123,6 +124,13 @@ int hal_timerRegister(int (*f)(unsigned int, cpu_context_t *, void *), void *dat
 }
 
 
+char *hal_timerFeatures(char *features, unsigned int len)
+{
+	hal_strncpy(features, "Using EPIT and GPT timers", len);
```

Comment:
For safety:
```suggestion
	hal_strncpy(features, "Using EPIT and GPT timers", len);
	features[len - 1] = '\0';
```
also in other functions.